cahiers-du-centre-gustave-glotz.csl

```diff
@@ -195,8 +195,19 @@
                 <text variable="container-title" form="short" font-style="italic"/>
                 <text variable="volume"/>
                 <text variable="issue"/>
-                <text macro="pages"/>
               </group>
+                <choose>
+                  <if variable="page" match="any">
+                    <text macro="pages"/>
+                  </if>
+                  <else>
+                    <group prefix=" [" suffix="]">
```

Comment:
you'll want to test for `<if variable="URL">` here to be safe. It's rare but possible that we have items with access date but no URL.